tests/test_data_objects.py

```diff
@@ -270,22 +270,32 @@ def test_to_qgis3d(self, borehole_data):
         assert outfile.is_file()
         outfile.unlink()
 
+    @pytest.mark.unittest
+    def test_to_kingdom(self, borehole_data):
```

Comment:
Er worden wat situaties gemist door de if/else statements in de export functie. Zou je die ook nog kunnen testen?